models/groups/map_developers.yml

```diff
@@ -159,20 +153,9 @@ minecraft_permissions:
   - setting.nick.set
   - setting.nick.unlimited
   - setting.nick.view
-  - sudo
   - whitelist.bypass
-  - whitelist.edit
-  - worldedit.*
-  - worldedit.analysis.*
-  - worldedit.clipboard.*
   - worldedit.drain
   - worldedit.extinguish
-  - worldedit.history.*
-  - worldedit.navigation.*
-  - worldedit.region.*
-  - worldedit.remove
-  - worldedit.selection.*
-  - worldedit.tool.info
   - worldedit.wand
```

Comment:
These changes would give mapdevs a wand but not let them used it for anything. Drain and extinguish don't require a wand, so you should probably just remove the wand permissions